src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -834,8 +834,8 @@ void GcsActorManager::Initialize(const GcsInitData &gcs_init_data) {
   // Notify raylets to release unused workers.
   gcs_actor_scheduler_->ReleaseUnusedWorkers(node_to_workers);
 
-  RAY_LOG(DEBUG) << "The number of registered actors is " << registered_actors_.size()
-                 << ", and the number of created actors is " << created_actors_.size();
+  RAY_LOG(INFO) << "The number of registered actors is " << registered_actors_.size()
```

Comment:
Instead of changing this to INFO, can we have more generalized log messages in the initialize step? Like we are doing it for DebugString